src/basic/linux/if_link.h

```diff
@@ -588,6 +588,8 @@ enum {
 	IFLA_MACVLAN_MACADDR,
 	IFLA_MACVLAN_MACADDR_DATA,
 	IFLA_MACVLAN_MACADDR_COUNT,
+    IFLA_MACVLAN_BC_QUEUE_LEN,
+    IFLA_MACVLAN_BC_QUEUE_LEN_USED,
```

Comment:
nit: indentation seems off

Reply:
Please try to use `src/basic/linux/update.sh`.